coremltools/models/neural_network/quantization_utils.py

```diff
@@ -752,7 +754,12 @@ def _lstmwp_to_fp16_lstmwp(lstm_wp, nbits, qm, i_size, o_size, has_peephole=True
             raise Exception('Unknown layer ' + layer_type + ' to be quantized')
 
 
+@deprecated(str="Instead use 'quantize_weights'")
```

Comment:
The string here is going to be concatenated with the standard message plus a semicolon. So you probably don't want a capital "I" and you probably want to end it in period.

Reply:
updated